modules/ROOT/pages/afm-modify-monitor.adoc

```diff
@@ -0,0 +1,42 @@
+= Modify a Monitor
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+You can modify monitors via the Functional Monitoring UI, which includes the options to edit the monitor as text fields or to edit the monitor as BAT source. Additionally you can download the monitor as BAT source, edit the monitor in your text editor of choice, and then upload the monitor.
+
+This procedure describes modifying monitors via the Functional Monitoring UI.
+
+== Procedure
+
+There are two methods to modify a monitor.
+
+To modify a monitor from the homepage:
+
+. Hover over the monitor to modify. 
+. Click *Edit* (image:afm-ui-edit-button.png[width=1.5%,height=1.5%]).
+
+To modify a monitor from its detail page:
+
+. Select the monitor that you want to modify.
+. Click the monitor.
+. Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) next to the monitor name.
+. Tap *Edit Monitor*.
+
+== Result
+
+The following options appear:
```

Comment:
Is this the result of both methods?

Reply:
And do we need to specify where they appear? Just in the same screen as the steps?

Reply:
I changed it to reflect that the options appear after following either method. I'm thinking we don't need to specify where they appear, as they occupy most of the UI real estate.